README.md

```diff
@@ -88,30 +79,28 @@ Bundling your application with webpack is awesome. What's less awesome is having
 Add these scripts to your `package.json`:
 
 ```json
-"start": "UV_THREADPOOL_SIZE=20 scriptoni web-dev-ts -c ./config",
-"build": "UV_THREADPOOL_SIZE=20 scriptoni web-build-ts -c ./config"
+"start": "UV_THREADPOOL_SIZE=20 scriptoni web-dev -c ./config",
+"build": "UV_THREADPOOL_SIZE=20 scriptoni web-build -c ./config"
 ```
 
 where:
 
 - the `UV_THREADPOOL_SIZE` trick is a workaround for a known issue with the sass-loader (https://github.com/webpack-contrib/sass-loader/issues/100). *You'll need this only if your project has more than a few `.sass` files*
-- the `-c ./config` points to a directory containing configuration for your project (read more below).
-
-**config dir (WIP)**
+- the `-c ./config` points to a **directory** containing the configurations for your project (read more below).
 
-*This "API" is very work in progress at the moment*
+**`./config` directory**
 
-The config dir for a project should include:
-- a `Config.js` file. It should export a tcomb type validating the configuration. Currently only `port` is strictly required by scriptoni webpack to work
-- any of `production.json`, `development.json`, `local.json` (all are optional): production and development should be tracked in version control, they are the default/base for `NODE_ENV=production` and `=development`, respectively. `local.json` is inteded to be used for custom, per-developer config tweaks, and should not be tracked.
+The `config` directory should include:
+- a `Config.js` file. It should export a `tcomb` type validating the configuration. Currently only `port` is strictly required by scriptoni webpack to work
```

Comment:
is this still a tcomb type? Or an io-ts one now?

Reply:
`io-ts`, I'll fix the readme